### PR TITLE
Addition to Whitepaper Link

### PR DESCRIPTION
**Pull request by joinimran**
Added updated link for white paper.
problem: https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/721

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/office-365-data-immutability.md [hunk 1]
```diff
@@ -53,4 +53,4 @@ Specifically:
 - Archiving in Office 365 allows users to combine different retention policies and hold actions to create granular retention policies. THese policies define the type or location of the items preserved and the duration of preservation.
```
Comment (JohanFreelancer9) [inline Enterprise/office-365-data-immutability.md]:
```suggestion
- Archiving in Office 365 allows users to combine different retention policies and hold actions to create granular retention policies. These policies define the type or location of the items preserved and the duration of preservation.
```

**Comment by joinimran** [comment]
@mypil - Ready to go live

File: Enterprise/office-365-data-immutability.md [hunk 2]
```diff
@@ -53,4 +53,4 @@ Specifically:
...
 
-To understand how Office 365 helps you meet regulatory obligations, specifically in relation to Rule 17a-4 requirements, see this [whitepaper](https://go.microsoft.com/fwlink/?linkid=830440) that covers Exchange Online Archiving, SharePoint Online, OneDrive for Business, and Skype for Business. The whitepaper also provides an in-depth analysis of Office 365 archiving features and functionalities against each of the requirements under SEC Rule 17a-4 and demonstrates to regulated customers how Office 365 archiving can enable them to meet these requirements.
+To understand how Office 365 helps you meet regulatory obligations, specifically in relation to Rule 17a-4 requirements, see  [whitepaper](https://www.microsoft.com/microsoft-365/blog/wp-content/uploads/2015/11/Microsoft-EOA-White-Paper.pdf) that covers Exchange Online Archiving, SharePoint Online, OneDrive for Business, and Skype for Business. The whitepaper also provides an in-depth analysis of Office 365 archiving features and functionalities against each of the requirements under SEC Rule 17a-4 and demonstrates to regulated customers how Office 365 archiving can enable them to meet these requirements.
```
Comment (JohanFreelancer9) [inline Enterprise/office-365-data-immutability.md]:
```suggestion
To understand how Office 365 helps you meet regulatory obligations, specifically in relation to Rule 17a-4 requirements, see the [whitepaper](https://www.microsoft.com/microsoft-365/blog/wp-content/uploads/2015/11/Microsoft-EOA-White-Paper.pdf) that covers Exchange Online Archiving, SharePoint Online, OneDrive for Business, and Skype for Business. The whitepaper also provides an in-depth analysis of Office 365 archiving features and functionalities against each of the requirements under SEC Rule 17a-4 and demonstrates to regulated customers how Office 365 archiving can enable them to meet these requirements.
```

**Comment by joinimran** [comment]
@mypil - Ready to go live

**Comment by mypil** [comment]
@robmazz - PR has been copy edited and is ready for final review. This is based on issue

Thank you.

cc: @kenwith

**Pull request merged**